SpellDataDump/spelldatadump_ptr.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/env bash
```

Comment:
@Dorovon I hope you don't mind me adding this file. It's PTR version of the `SpellDataDump/spelldatadump.sh`.

Reply:
This seems like a good idea to add, but just for consistency with the other version I think the shebang line should be the same.
```suggestion
#!/bin/sh
```

Reply:
I think that shebang needs to change in `spelldatadump.sh` because `sh` doesn't support arrays. I couldn't get `spelldatadump.sh` to run on my machine, it was reporting `Syntax error: "(" unexpected` with `#!/bin/sh.` `#!/bin/env bash` or `#!/bin/bash` seems to work just fine.

Reply:
Oh, that's interesting; I hadn't noticed problems but maybe I've just been running it with `bash` without thinking. I guess it makes sense to change them to #!/bin/bash.

Reply:
sh may or may not support arrays, depending on the actual shell it is (distribution dependant, somewhat). Using bash here is probably sensible.